README.md

```diff
@@ -1,14 +1,56 @@
 
-The Media for Edge Network extension is currently in Beta. Use of this code is by invitation only and not otherwise supported by Adobe. Please contact your Adobe Customer Success Manager to learn more.
+## About this project
 
-By using the Beta, you hereby acknowledge that the Beta is provided "as is" without warranty of any kind. Adobe shall have no obligation to maintain, correct, update, change, modify or otherwise support the Beta. You are advised to use caution and not to rely in any way on the correct functioning or performance of such Beta and/or accompanying materials.
+The Adobe Streaming Media for Edge Network extension sends data about audio and video consumption on your streaming applications to the Adobe Experience Platform Edge Network. This enables capabilities for measurement, analysis, and activation with media data across the Adobe Experience Cloud solutions when using the [Adobe Experience Platform Mobile SDK](https://developer.adobe.com/client-sdks) and the Edge Network extension.
+
+## Installation
+
+To install and start using the Media for Edge Network extension, check out the [getting started guide](Documentation/getting-started.md) and the [API reference](Documentation/api-reference.md).
+
+## Migrating from Media Analytics
+
+Please refer to the [Migrating from AEPMedia to AEPEdgeMedia](Documentation/migration-guide.md) guide.
+
+### Development
+
+**Open the project**
+
+To open and run the project, open the `code/settings.gradle` file in Android Studio
+
+### Development
+
+#### Run the test application
+
+To configure and run the test app for this project, follow the [getting started guide for the test app](Documentation/getting-started-test-app.md).
+
+#### Code format
+
+This project uses the code formatting tools [Spotless](https://github.com/diffplug/spotless/tree/main/plugin-gradle) with [Prettier](https://prettier.io/). Formatting is applied when the project is built from Gradle and is checked when changes are submitted to the CI build system.
+
+Prettier requires [Node version](https://nodejs.org/en/download/releases/) 10+
+To enable the Git pre-commit hook to apply code formatting on each commit, run the following to update the project's git config `core.hooksPath`:
+```
+make init
+```
+
+## Related Projects
+
+| Project                                                      | Description                                                  |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [Core extensions](https://github.com/adobe/aepsdk-core-android)                                    | The Mobile Core represents the foundation of the Adobe Experience Platform Mobile SDK. |
+| [Edge Network extension](https://github.com/adobe/aepsdk-edge-android) | The Edge Network extension allows you to send data to the Adobe Edge Network from a mobile application. |
```

Comment:
can you also include EdgeIdentity
| [Identity for Edge Network extension](https://github.com/adobe/aepsdk-edgeidentity-android) | The Identity for Edge Network extension enables handling of user identity data from a mobile app when using the Edge Network extension. |